### PR TITLE
[SITE-2496] Add workflow to maintain D7ES module

### PR DESCRIPTION
**Pull request by pwtyler**
Adds a CI workflow to update d7es module. This will need to be squashed into https://github.com/pantheon-systems/drops-7/commit/03af2f46cd8917171720314eea612a86ea74915a rather than being merged.

- [x] Test in fork
- [x] Determine any changes to URLs (now or later)
- [x] Squash into CI commit

https://getpantheon.atlassian.net/browse/SITE-2496
https://getpantheon.atlassian.net/browse/CCB-1303

### CODE REVIEW

**Comment by pwtyler** [comment]
Tested with https://github.com/pwtyler/drops-7/pull/1

@greg-1-anderson, once reviewed, I will squash into the canonical CI commit and push to `default`.

**Review by greg-1-anderson — APPROVED** [review]
LGTM, but let's wait for confirmation from Tag1 on where updates will be published.

**Comment by pwtyler** [comment]
The actual changes added and reviewed are the addition of the following files:
https://github.com/pantheon-systems/drops-7/pull/209/files#diff-4bea8c1f4b13cd4b2a0079b54d1b298b7c0640ad90c76e9fcb486d8eaa5a6c6f
https://github.com/pantheon-systems/drops-7/pull/209/files#diff-31eed2dcac36d584a06e84c70d77c3549781252cf1cb667d4b80b88119a846ee

This branch will need be force pushed to `default` so that we have a single CI commit to be removed in the course of regular releases.

File: .circleci/confirm-safety.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+#
+# The purpose of this script is to examine the base branch that this PR is
+# set to merge into by usig the GitHub API. We are only querying a public
+# repo here, so we do not need to use the GITHUB_TOKEN.
```
[the remark this answers is not in the document]
Comment (pwtyler) [reply .circleci/confirm-safety.sh:6]: This is a comment, Wiz.

**Pull request merged**